extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -101,7 +101,9 @@ define('ext.wikia.adEngine.adContext', [
 		var hasFeaturedVideo = context.targeting.hasFeaturedVideo;
 
 		context.bidders.prebid = !areDelayServicesBlocked() && isEnabled('wgAdDriverPrebidBidderCountries');
-		context.bidders.prebidAE3 = context.targeting.skin === 'oasis' && isEnabled('wgAdDriverPrebidAdEngine3Countries');
+		// Due to TOP_BOXAD rename we have to force new Prebid - otherwise it won't bid for this slot
+		// context.bidders.prebidAE3 = context.targeting.skin === 'oasis' && isEnabled('wgAdDriverPrebidAdEngine3Countries');
+		context.bidders.prebidAE3 = true;
```

Comment:
what about mobile-wiki?

Reply:
On mobile-wiki we use AE3 so it's not an issue I guess?